cpu/stm32/stmclk/stmclk_l4wx.c

```diff
@@ -643,5 +643,14 @@ void stmclk_init_sysclk(void)
         gpio_init_af(GPIO_PIN(PORT_A, 8), GPIO_AF0);
     }
 
+#if IS_USED(MODULE_PERIPH_USBDEV)
+    /* USBDEV is clocked by HSI48 so enable this clock when the peripheral is used */
+    RCC->APB1RSTR1 |= RCC_APB1RSTR1_USBRST;
+    RCC->CRRCR |= RCC_CRRCR_HSI48ON;
+    RCC->BDCR |= 1;
```

Comment:
Why do you need to force LSEON in the backup domain control register ?

Reply:
Probably a leftover. I had a hard time properly configure the clock for this driver back then.

Reply:
Not needed, removed.